docs/connect.html

```diff
@@ -103,7 +103,7 @@ <h4><a id="connect_configuring" href="#connect_configuring">Configuring Connecto
         <li><code>topics.regex</code> - A Java regular expression of topics to use as input for this connector</li>
     </ul>
 
-    <p>For any other options, you should consult the documentation for the connector.</p>
+    <p>For any other options, you should consult the <a href="#connectconfigs">documentation</a> for the connector.</p>
```

Comment:
Add link to the connector config section.
before:
![image](https://user-images.githubusercontent.com/43372967/82001955-0bc26380-968f-11ea-8271-08372e59c5b5.png)